src/librtlsdr.c

```diff
@@ -2186,7 +2209,7 @@ int rtlsdr_read_async(rtlsdr_dev_t *dev, rtlsdr_read_async_cb_t cb, void *ctx,
 	#ifdef _ENABLE_RPC
 	if (rtlsdr_rpc_is_enabled())
```

Comment:
If `bufs` is `NULL` then we are just called by `rtlsdr_read_async` and the RPC should work the same way as before, should not?